pom.xml

```diff
@@ -1110,6 +1111,8 @@
                                             <exclude>jakarta.xml.ws:jakarta.xml.ws-api:(, 3.0.1)</exclude>
                                             <exclude>jakarta.xml.soap:jakarta.xml.soap-api:(, 2.0.1)</exclude>
                                             <exclude>jakarta.jws:jakarta.jws-api:(, 3.0.0)</exclude>
+                                            <exclude>org.bouncycastle:bcprov-jdk15on</exclude>
```

Comment:
What do you think about excluding **everything** from `org.bouncycastle` and allowing only selected components. It would allow us to fail te build if some dependency brings something like `org.bouncycastle:bcprov-jdk16`

Reply:
Updated according to your suggestion.